src/AutoRest.CSharp/Common/Input/Configuration.cs

```diff
@@ -366,7 +371,7 @@ public static void Initialize(IPluginCommunication autoRest, string defaultNames
                 methodsToKeepClientDefaultValue: autoRest.GetValue<string[]?>(Options.MethodsToKeepClientDefaultValue).GetAwaiter().GetResult() ?? Array.Empty<string>(),
                 mgmtConfiguration: MgmtConfiguration.GetConfiguration(autoRest),
                 mgmtTestConfiguration: MgmtTestConfiguration.GetConfiguration(autoRest),
-                branded: GetOptionBoolValue(autoRest, Options.Branded),
```

Comment:
Can we keep branded for now?
'Don't remove yet currently supported flag branded to help the transition to the new system. If both flags are set, read flavor first and ignore branded boolean flag.'

Reply:
> Can we keep branded for now? 'Don't remove yet currently supported flag branded to help the transition to the new system. If both flags are set, read flavor first and ignore branded boolean flag.'

He has changed the requirement, but the issue description is not updated.